src/benchmark/file_based_benchmark.cpp

```diff
@@ -23,8 +23,8 @@ int main(int argc, char* argv[]) {
 
   // clang-format off
   cli_options.add_options()
-      ("table_path", "Directory containing the Tables", cxxopts::value<std::string>()->default_value("")) // NOLINT
-      ("query_path", "Directory/file containing the queries", cxxopts::value<std::string>()->default_value("")) // NOLINT
+      ("table_path", "Directory containing the Tables as csv, tbl or binary files. CSV files require meta-files, see csv_meta.hpp or any *.csv.json file.", cxxopts::value<std::string>()->default_value(".")) // NOLINT
+      ("query_path", "Directory containing .sql files or a specific .sql file", cxxopts::value<std::string>()->default_value(".")) // NOLINT
```

Comment:
Mmh, "directory containing ... a specific file"? Maybe flip to remove ambiguity? "Single file or dir"